### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by dns2utf8**
started a README.md for the lazy builder.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+
+# Install Guide Arch Linux
+
+    sudo pacman -S hiredis libev cmake boost
+    git clone --recursive https://github.com/fmorgner/webdev.git
+    cd webdev/cpp/build/
```
Comment (fmorgner) [inline README.md]: no. cd webdev

File: README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    cmake ..
```
Comment (fmorgner) [inline README.md]: no. ./cpp/build/build

File: README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+    ./build
```
Comment (fmorgner) [inline README.md]: no. already done above

File: README.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+
+
+# Install Guide Arch Linux
+
+    sudo pacman -S hiredis libev cmake boost
```
Comment (fmorgner) [inline README.md]: add jsoncpp

File: README.md (hunk 2 above)
Comment (fmorgner) [inline README.md]: Maybar add -r to build release only

**Comment by dns2utf8** [comment]
Fixed it, getting this:

```
/include/resources/resource_frontend.h:9:10: fatal error: 'httpserver.hpp' file not found
```

**Comment by fmorgner** [comment]
you need to install aur/libhttpserver-git

**Pull request merged**